### DISCUSSION TITLE
May I make a written guide for UABEA in my native language?

### DISCUSSION THREAD

**Discussion by enarkay** [nesrak1/UABEA#225] [General]
Before I start - I am terribly sorry if this isn't the right place to ask.
In such case, please be so kind as to redirect me to where I should be asking instead.

Now if I may ask, I would like to ask if you're okay with me making a guide on how to use UABEA in my native language (Korean).

There are so many people in my country's game localization community,
who are very interested in using the software but are having a hard time learning how to use it.
There unfortunately isn't really a lot of kind/detailed learning documents that's written in Korean, for beginners just starting out.

However, there is a guy in our game localization community who claims that when he asked for your permission to make a guide on how to use UABEA, you refused.

I simply don't understand why you would refuse anyone to make a guide on how to use your software,
which you obviously created and made public for people to use.

So I believe he is either lying or misunderstood you somehow, but wanted to ask you to be 100% sure.

Question 1) So, are you okay with me making a guide for UABEA?
Question 2) If not, then could you be so kind as to explain why you DON'T want me to make a guide?

Thank you.

**Comment by nesrak1** [comment]
I have no issues with people making guides about UABEA (that would be weird if I did), and I'm not sure what person asked me and thought I refused them, but I did not do that.

**Reply by enarkay** [reply]
Just as I thought - thank you for such a prompt reply and confirmation, as well as huge thanks for such an amazing software and the time and efforts you invest into maintaining it :)

**Comment by rxnnxs** [comment]
I do not even find the guide at all.
So who can tell me the function of the mod maker or the packager (compress) or where to find a short excplanation to the functions?

**Reply by enarkay** [reply]
I believe the reason there is no written guide is that nesrak did not really care much about the mod maker feature in the first place, because he was/is planning to replace it with a much better feature than the current one in the future. Or that's what I believe I've heard. But I've used it enough times to be able to explain to someone how to use it. Leave your discord and I'll contact and tell you.

**Reply by rxnnxs** [reply]
What do you mean with leave my discord?
Thank you for your offered help.
When I am changing some assets, I ask myself if I can overload them. But I guess I can not, it has to be in an asset.
So if a new update overwrites my changed contents inside an asset, my work has to be redone.
So I wonder if I can create a "patch" or save only my changes to that asset and then load it up to the new asset (which was updated by the developer) and overwrite again those assets I have changed with my worlk.

How do you do this?

**Reply by enarkay** [reply]
Well I didn't know someone could have trouble interpreting what it means.
It literally means as is written - leave your discord (give me your discord id), and I'll contact you on discord.

To explain the steps and processes of what you want will most likely require a lot of questions and answers back and forth, and probably not just texts but also screenshots. All this would most definitely be easier, faster, and more efficient if we talked on discord, instead of creating multiple posts here.
